pkg/apiserver/registry/datapackaging/package_crd_rest.go

```diff
@@ -340,6 +347,7 @@ func (r *PackageCRDREST) ConvertToTable(ctx context.Context, obj runtime.Object,
 			{Name: "PackageMetadata Name", Type: "string", Format: "name", Description: "Associated PackageMetadata name"},
 			{Name: "Version", Type: "string", Description: "Version"},
 			{Name: "Age", Type: "date", Description: "Time since resource creation"},
+			{Name: "Included Software", Type: "array", Description: "Software Included in Package"},
```

Comment:
i dont think we need to show this info in the table:
- it becomes too wide for most screens
- most packages cover only single piece of software so package version is a good proxy (details can always be obtained via kubctl get or eventually in kctrl)

Reply:
I agree, leaving this comment open for @seemiller to take a look. But I have reverted the change for now.